cl/tests/base.py

```diff
@@ -94,13 +108,13 @@ def _update_index():
         # For now, until some model/api issues are worked out for Audio
         # objects, we'll avoid using the cl_update_index command and do
         # this the hard way using tasks
-        keys = [opinion.pk for opinion in Opinion.objects.all()]
-        add_or_update_opinions(keys)
-
-
+        opinion_keys = [opinion.pk for opinion in Opinion.objects.all()]
```

Comment:
This line can probably be:

    opinion_keys = Opinions.object.values_list('pk', flat=True)

Ditto below too.